webapp/src/dialogs.tsx

```diff
@@ -168,6 +165,38 @@ export function showAboutDialogAsync() {
     }).done();
 }
 
+
+function renderCompileLink(cs: pxt.TargetCompileService) {
+    let url: string;
+    let version: string;
+    let name: string;
+
+    if (typeof cs.codalTarget === "object" && typeof cs.codalTarget.url === "string") {
+        // There is also a branch in this object but looks like it is sometimes a branch and sometimes a tag
+        // so can't really generate a good URL
```

Comment:
Assume `branch` is in fact a tag. Assume it's always there. It can be branch on non-production targets, but we don't care in that case. Don't take the `cs.gittag` - this is a version of the "build system", just default to `"master"` or something.

Reply:
I mean, build the link as if `branch` was always a tag. If it's a branch it messes up caching, so even during development it's almost always a tag. We don't care if the link is broken otherwise.